### PR TITLE
Update bglibpy version

### PR DESCRIPTION
**Pull request by wvangeit**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-bglibpy/package.py [hunk 1]
```diff
@@ -14,6 +14,7 @@ class PyBglibpy(PythonPackage):
 
     version('develop', branch='master')
 
+    version('4.3.19', commit='bce00a1ddedb605a5ed5225989192eb5f7e133ae')
```
Comment (pramodk) [inline var/spack/repos/builtin/packages/py-bglibpy/package.py:17]: for module purpose we also have entry here: https://github.com/BlueBrain/spack/blob/2c8bdeeff148c28fb280adfd9bf7ecc518153012/deploy/packages/bbp-packages.yaml#L126

could you also update that one?

(By the way - CI will run after bb5 maintenance)

Comment (wvangeit) [reply var/spack/repos/builtin/packages/py-bglibpy/package.py:17]: Ok, done.

This process urgently needs to be automated. I'll definitely work on BGLibPy automating a PR when a new version is released.
Is there maybe any plan to e.g. let spack automatically add new python packages when they're released to devpi?

Comment (wvangeit) [reply var/spack/repos/builtin/packages/py-bglibpy/package.py:17]: (Btw, about this bbp-packages file? Is there way to specify that it should automatically pick the latest version?)

Comment (matz-e) [reply var/spack/repos/builtin/packages/py-bglibpy/package.py:17]: @wvangeit with the current setup, dropping the version number from `bbp-packages.yaml` will keep modules for all versions of `py-bglibpy` in the deployment archives.

I have to test it still, but in the next deployment iteration, we should be able to drop the version specification and just list software to be installed.

**Review by anilbey — APPROVED** [review]

**Comment by wvangeit** [comment]
@anilbey we already have a new version by now :-) Could you create another one for version 4.4? Thx.

**Comment by anilbey** [comment]
Here it is #920

**Pull request merged**